### PR TITLE
Improve developer experience

### PR DESCRIPTION
**Pull request by aspacca**

### CODE REVIEW

**Comment by aspacca** [comment]
rationale: separate docker run for CI and local run for developer that doesn't require build the docker image

File: tests/scripts/flake8.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -e
+
+flake8 .
```
Comment (v1v) [inline tests/scripts/flake8.sh:5]: NIT: if this script is only calling `flake8 .` what do you think to remove it and add the call in the above consumers?

File: tests/scripts/mypy.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -e
+
+mypy --install-types --non-interactive .
```
Comment (v1v) [inline tests/scripts/mypy.sh:5]: NIT: if this script is only calling `mypy` what do you think to remove it and add the call in the above consumers?

**Review by v1v — APPROVED** [review]
LGTM, some minor comments about removing a couple of files in favour or calling the command itself

File: tests/scripts/flake8.sh (hunk 1 above)
Comment (aspacca) [reply tests/scripts/flake8.sh:5]: this way I can run the same script from docker or on the host.
If I move in the docker script then I cannot use the same script for both

File: tests/scripts/mypy.sh (hunk 2 above)
Comment (aspacca) [reply tests/scripts/mypy.sh:5]: same as  above

File: tests/scripts/flake8.sh (hunk 1 above)
Comment (aspacca) [reply tests/scripts/flake8.sh:5]: we could have in the consumer script the call to `flake8 .`, but still I'd need this script for running on the host and I have to maintain eventual changes in both place

**Pull request merged**